### ISSUE TITLE
Destiny 2 Unpacker Issue

### ISSUE THREAD

**Issue by kloruklass** [MontagueM/DestinyUnpackerCPP#1]
Can you make it more accessible for newbies? I have the game, but I don't know how to edit the code.
Plus, I don't have Visual Studio or other software that you probably need to compile and run it.

Maybe something with TXT files, where you can easily edit the path or other specific lines.
I'm trying to unpack Destiny 2 for too long now, and nothing worked.

There are like 2.205 PKG files. I was hoping for something that would extract them automatically.

Looking forward to your reply.

**Comment by kloruklass** [comment]
It creates a "test_out" folder but it's completely empty.

**Comment by C0D3-M4513R** [comment]
Did you do this? `Put oodle dll in same directory. The oodle dll is from Destiny 2 directory.`
Is the Steam path correct in the Software? 


For automatically extracting stuff: Don't extract everything at once. That is a TON of data, and unnessesary.

**Comment by kloruklass** [comment]
Yes, I tried everything, literally, and yet nothing worked so far.
I wonder if someone has the same problem.

**Comment by kloruklass** [comment]
I don't know the reason, but I think it might have something to do with the game version.
If the developer could give out some more info it would be nice.

**Comment by C0D3-M4513R** [comment]
What is the output of the exe? Did you even get this repo to compile?
What exactly did you try. Trying eveything also would mean diving into compilation errors and runtime errors. This repo works.

Since you have the full source, you don't need more information. Just read the source code.

**Comment by C0D3-M4513R** [comment]
https://github.com/MontagueM/DestinyUnpackerCPP/blob/dec65ae0c6d95b6855f5bb23bbdfe28561dacd26/package.h#L82
https://github.com/MontagueM/DestinyUnpackerCPP/blob/dec65ae0c6d95b6855f5bb23bbdfe28561dacd26/main.cpp#L7
https://github.com/MontagueM/DestinyUnpackerCPP/blob/dec65ae0c6d95b6855f5bb23bbdfe28561dacd26/package.h#L67
